src/components/v2/VaultItemUi/index.tsx

```diff
@@ -0,0 +1,132 @@
+// VaultItemUi
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { useTranslation } from 'translation';
+import { formatCommaThousandsPeriodDecimal, formatToReadablePercentage } from 'utilities/common';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { useStyles } from './styles';
+
+type VaultAssetType = 'vai' | 'xvs' | 'vrt';
+
+export interface IVaultItemUiProps {
+  stakeTokenName: VaultAssetType;
+  earnTokenName: VaultAssetType;
+  rewardValue?: number;
+  stakingValue: number;
+  stakingAprValue: number;
+  dailyEmissionValue: number;
+  totalStakedValue: number;
+  onClaim: () => void;
+  onStake: () => void;
+  onReward: () => void;
+  className?: string;
+}
+
+export const VaultItemUi = ({
+  stakeTokenName,
+  earnTokenName,
+  rewardValue,
+  stakingValue,
+  stakingAprValue,
+  dailyEmissionValue,
+  totalStakedValue,
+  onClaim,
+  onStake,
+  onReward,
+  className,
+}: IVaultItemUiProps) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const dataListItems = useMemo(
+    () => [
+      {
+        title: t('vaultItemUi.stakingApr', { stakeTokenName: stakeTokenName.toUpperCase() }),
+        value: formatToReadablePercentage(stakingAprValue),
+      },
+      {
+        title: t('vaultItemUi.dailyEmission'),
+        value: (
+          <>
+            <Icon css={classes.tokenIcon} name={earnTokenName} />
+            {formatCommaThousandsPeriodDecimal(dailyEmissionValue)}
+          </>
+        ),
+      },
+      {
+        title: t('vaultItemUi.totalStaked'),
+        value: (
+          <>
+            <Icon css={classes.tokenIcon} name={stakeTokenName} />
+            {formatCommaThousandsPeriodDecimal(totalStakedValue)}
+          </>
+        ),
+      },
+    ],
+    [stakeTokenName, earnTokenName, stakingAprValue, dailyEmissionValue, totalStakedValue],
+  );
+
+  return (
+    <Paper css={classes.container} className={className}>
+      <div css={classes.header}>
+        <div css={classes.title}>
+          <Icon css={classes.tokenIcon} name={stakeTokenName} />
+          <Typography variant="h4" css={classes.text}>
+            {stakeTokenName.toUpperCase()}
```

Comment:
We should use `getToken` with the token ID passed through props (see previous comment) so that we can retrieve the token data and render its symbol here.

Reply:
done